index.html

```diff
@@ -975,6 +980,86 @@ <h2>Methods</h2>
             selected candidate pair =].
           </p>
         </dd>
+        <dt>
+          <dfn>removeCandidatePairs</dfn>
+        </dt>
+        <dd>
+          <p>
+            The {{removeCandidatePairs}} method removes the provided candidate pairs, making them unavailable to use
```

Comment:
What's the use case for this method? What happens when the selected pair is removed?

Reply:
Clarified the effects and intended usage of the method.